Documentation/Signoff/LCD Block Subsystem.md

```diff
@@ -0,0 +1,110 @@
+# LCD Block Subsystem
+
+## Subsystem Function:
+
+The purpose of this subsystem is to help ease the user's understanding of the kit's functionalities, provide a digital user manual, provide the users with instructions or guided learning via challenges and lessons and enable a digital interaction with the car. Hence, this LCD Block Subsystem serves as the one of the primary interaction point between the user and the device as a whole. 
+
+## Constraints/Specifications:
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility indoor and outdoor with a brightness level of at least 350 nits.                   |  Interaction Design principles [3]             |
+| User Interface    | The menu should allow easy navigation at any point with buttons such as "Return to Menu" or "Next" or "Back".    | User interface design [4]       |
+| Interactivity     | The system must register input from the keypad within 300ms to ensure responsive interaction.        | Performance Requirement [6]        |
+| environmental and social responsibility      |  The system must contain, on average, a minimum of 2% of any combination of postconsumer recycled plastic, ITE-derived post-consumer recycled plastic, or bio based plastic, measured as a percentage of total amount of plastic (by weight) in the product. | IEEE Standard [5] |
+
+## Schematic:
+
+<p align="center">
+  <img height="500" src="https://github.com/user-attachments/assets/c6c65c65-3b30-4d90-81c9-5c3017dd02a9">
```

Comment:
What does the power switch do? Is it wired correctly?

Should Vcc on the ultrasonic module be connected to a processor GPIO pin?